services/admin.py

```diff
@@ -1,5 +1,6 @@
 from adminsortable.admin import SortableAdmin
 from django.contrib import admin
+from django.utils.translation import ugettext_lazy as _
```

Comment:
The prefixed gettext functions will likely be deprecated in the future. I think we should just use "gettext*" without the u everywhere.

Reply:
Agree.